common/djangoapps/course_modes/admin.py

```diff
@@ -1,32 +1,19 @@
 """
 Django admin page for course modes
 """
-from django.conf import settings
 from django import forms
-from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
 from django.contrib import admin
-
-from pytz import timezone, UTC
-
+from django.utils.translation import ugettext_lazy as _
+from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
-from opaque_keys import InvalidKeyError
+from pytz import UTC, timezone
 
-from util.date_utils import get_time_display
-from xmodule.modulestore.django import modulestore
 from course_modes.models import CourseMode, CourseModeExpirationConfig
-
-# Technically, we shouldn't be doing this, since verify_student is defined
```

Comment:
Good catch, @adampalay. Unfortunately isort often drops comments on the floor, so we're going to have to keep an eye out. I'll revert this change.